test/Datadog.Trace.ClrProfiler.IntegrationTests/Helpers/SpanTestHelpers.cs

```diff
@@ -6,11 +6,12 @@
 
 namespace Datadog.Trace.ClrProfiler.IntegrationTests
 {
-    public class WebServerTestHelpers
+    public class SpanTestHelpers
     {
-        public static void AssertExpectationsMet(
-            List<WebServerSpanExpectation> expectations,
+        public static void AssertExpectationsMet<T>(
+            List<T> expectations,
```

Comment:
nit: Could this just be `List<SpanExpectation>`?

Reply:
It could, but then every List of expectations would need to be an explicit list of `SpanExpectation`  and not the implementation. It's workable obviously, but it was less burden on the test class IMO. 
That way the list matches the implementation for that test and any extra properties or methods or whatever are just readily available to the test class.

Reply:
Hmm. Maybe I'm missing something, but I don't see what's wrong with making all of the lists be of the parent class `SpanExpectation` since this method is only operating on that API level. This is a small nit so I don't want to hold up the review on it, but if my thinking is correct, this could help reinforce a separation of concerns by making this helper always use the overarching `SpanExpectation` API and it's the responsibility of the subclasses to implement them correctly.